sdk/identity/identity/src/util/validateMultiTenant.browser.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GetTokenOptions } from "@azure/core-auth";
+
+/**
+ * @internal
+ */
+export const multiTenantErrorMessage =
```

Comment:
since variable doesn't seem to get used in the browser case, the message doesn't really make sense to me. Shouldn't it be talking about the tenantId being `adfs`?

Reply:
correct! My brain didn’t click